src/Components/Books.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+function Books() {
+  return (
+    <div>
+      <Navbar />
+      <div className="container">
+        <h2>List of availabe books!</h2>
```

Comment:
Kindly you need to render a list of books using a book component that takes the book data as a prop and renders the title and the author, Please fix this issue :) :)